src/miral/basic_window_manager.cpp

```diff
@@ -2864,14 +2869,22 @@ void miral::BasicWindowManager::update_application_zones_and_attached_windows()
                 case mir_window_state_attached:
                     if (info.exclusive_rect().is_set())
                     {
-                        first_pass.push_back(&info);
+                        // We are sorting east-to-west connected edges to the back of the list so that they
+                        // force everyone else out of the way when they're placed. By "east-to-west", we are
+                        // referring to "top" and "bottom" panels, as opposed to "left" and "right" panels.
```

Comment:
We have a comment above that agrees with the code, we don't need another here that disagrees